sources/lib/Resources/config/services/pomm.yml

```diff
@@ -22,6 +18,23 @@ services:
         tags:
             - { name: "property_info.type_extractor" }
 
+    pomm.session_builder:
+        class: 'PommProject\PommBundle\Model\SessionBuilder'
+        arguments: ['@?']
+
+    pomm.pooler.model:
```

Comment:
Ok, symfony also use the singular form (ie `twig.extension.debug`). That seems strange for me, but ok.